extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/indexing/MSQCompactionRunnerTest.java

```diff
@@ -85,6 +85,7 @@ public class MSQCompactionRunnerTest
 
   private static final String TIMESTAMP_COLUMN = "timestamp";
   private static final int TARGET_ROWS_PER_SEGMENT = 100000;
```

Comment:
is this field still used in the tests?

Reply:
Yes, for validating target is also a valid param.